attribution-graphs-frontend/addition/init-add-connections.js

```
@@ -0,0 +1,76 @@
+  var featureContainerSel = sel.append('div.feature-container')
+
+  renderAll.clickIdx.fns.push(() => {
+    var features = allFeatures.idxLookup[visState.clickIdx][type]
+      .map(d => ({...d, layer: allFeatures.idxLookup[d.idx].layer}))
+
+    var featureWidth = 100
+    var gap = 20
+
+    var availableWidth = (window.innerWidth - 320 - 90)/2 // middle col and gaps
+    var nCols = 2 * Math.max(1, Math.floor(availableWidth/(2*(featureWidth + gap)))) // even number directly, min 1 per side
+    var nCols = Math.max(4, nCols)
```

Biome (1.9.4)

[error] Shouldn't redeclare 'nCols'. Consider to delete it or rename it.

'nCols' is defined here:


(lint/suspicious/noRedeclare)